gemfile.d/foreman_bootdisk.rb

```diff
@@ -0,0 +1,5 @@
+# For testing the foreman bootdisk proxmox extensions
+# this gem file is required
+group :test do
```

Comment:
I think this file is just not loaded at all. Can you try puting the same code into `Gemfile`?

Reply:
Yeah I had the same comment above.

Reply:
It still skipped the tests.

Reply:
Right. Because it's stupid.

Can you try adding it here?
https://github.com/theforeman/foreman_bootdisk/blob/913d42b3c9bf032ac67cb7dde4c8230812b0b481/.github/workflows/ruby_ci.yaml#L52
so a second line, a la
```
           echo "gem 'foreman_fog_proxmox'" >> bundler.d/foreman_bootdisk.local.rb 
```